eZ/Publish/API/Repository/RoleService.php

```diff
@@ -338,6 +343,30 @@ public function assignRoleToUser(Role $role, User $user, RoleLimitation $roleLim
     public function unassignRoleFromUser(Role $role, User $user);
 
     /**
+     * Loads a user group role assignment for the given id.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If the role assignment, role or user group was not found
+     *
+     * @param mixed $assignmentId
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\UserGroupRoleAssignment
+     */
+    public function loadUserGroupRoleAssignment($assignmentId);
+
+    /**
+     * Loads a user role assignment for the given id.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If the role assignment, role or user was not found
+     *
+     * @param mixed $assignmentId
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\UserRoleAssignment
+     */
+    public function loadUserRoleAssignment($assignmentId);
```

Comment:
What happens when you try to load UserGroupRoleAssignment from here?
Actually, as the IDs for role assignments are "shared", I'm not convinced we need this together with `loadUserGroupRoleAssignment()`. One `loadRoleAssignment()` should be enough.